app/models/nextPayments/viewmodels/WYOClaimToAdjustViewModel.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.nextPayments.viewmodels
+
+import models.incomeSourceDetails.TaxYear
+
+case class WYOClaimToAdjustViewModel(adjustPaymentsOnAccountFSEnabled: Boolean,
```

Comment:
I dont think we need to pass this FS as a part of this view model, we should not be able to reach this part of evaluation/logic in the first place within a controller

Reply:
Should I pass the ViewModel (without the FS) to the View as an Option then?

Reply:
please ignore my comment, it should be fine in this case